spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,217 @@
+WIN_ISO_ROOT = 'https://artifactory.delivery.puppetlabs.net/artifactory/generic__iso/iso/windows'.freeze
+WIN_2012R2_ISO = 'en_windows_server_2012_r2_with_update_x64_dvd_6052708.iso'.freeze
+QA_RESOURCE_ROOT = 'https://artifactory.delivery.puppetlabs.net/artifactory/generic__iso/iso/SQLServer'.freeze
+SQL_2019_ISO = 'SQLServer2019CTP2.4-x64-ENU.iso'.freeze
+SQL_2017_ISO = 'SQLServer2017-x64-ENU.iso'.freeze
+SQL_2016_ISO = 'en_sql_server_2016_enterprise_with_service_pack_1_x64_dvd_9542382.iso'.freeze
+SQL_2014_ISO = 'SQLServer2014SP3-FullSlipstream-x64-ENU.iso'.freeze
+SQL_2012_ISO = 'SQLServer2012SP1-FullSlipstream-ENU-x64.iso'.freeze
+SQL_ADMIN_USER = 'sa'.freeze
+SQL_ADMIN_PASS = 'Pupp3t1@'.freeze
+
+RSpec.configure do |c|
+  c.before(:suite) do
+    run_shell('puppet module install puppetlabs/mount_iso')
```

Comment:
If you use the `configure!` functionality, you'll need to define a Helper singleton to access the Litmus bolt helpers outside of `it` blocks:

```ruby
require 'puppet_litmus'
require 'singleton'

class Helper
  include Singleton
  include PuppetLitmus
end
```

You would then access `run_shell` and other litmus helper methods like:

```ruby
Helper.instance.run_shell('foo')
```

- [Source](https://github.com/puppetlabs/puppetlabs-powershell/blob/f2acaee81110641575d3cc2cbc5c690cd7635fd3/spec/spec_helper_acceptance_local.rb)

Reply:
Updated